internal/server/data/migrations.go

```diff
@@ -714,3 +715,30 @@ func addProviderUserSCIMFields() *migrator.Migration {
 		},
 	}
 }
+
+func addDeviceFlowAuthRequestTable() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-10-06T14:58",
+		Migrate: func(tx migrator.DB) error {
+			_, err := tx.Exec(`
+				CREATE TABLE IF NOT EXISTS device_flow_auth_requests (
```

Comment:
We could add an org ID here from the initial request, could be nice where we are thinking of making org ID a standard thing

Reply:
in the future we want to be able to do device flow requests without knowing the org, so putting it here would be a problem